### PR TITLE
Fixes #55 Allow both circle and rect in display

### LINKED ISSUE #55: Use both rectangels and circles
I think it would be great to allow one to make a timeline with both rectangles and circles. That way events without a stop date could be displayed as a circle, whereas events with start and stop date could be a rectangle. Is this possible in the current implementation?

### PR DESCRIPTION
**Pull request by inodb**
Add feature to set display attribute for data points so one can display both
rectangles and circles in the same timeline.

### CODE REVIEW

**Comment by inodb** [comment]
Let me know if you want me to make a new example in ``examples/examples.html`` or if there's anything else I should change

**Comment by jiahuang** [comment]
Could you edit the readme to reflect this change as well as add a new example?

**Comment by inodb** [comment]
I added an example. Only problem is the positioning of the circles compared to the rectangles. The rect and circle combo example works for both stacked view and single track. It unfortunately breaks the only circle example. As can be seen in: 

screen shot 2015-05-21 at 12 04 11 pm

Do you have any suggestions?

**Comment by inodb** [comment]
Fixed it. I'll update the README.

screen shot 2015-05-21 at 2 23 26 pm

**Comment by inodb** [comment]
Updated the README with the rect/circle example and changed the image sources to use relative URLs

File: README.md [hunk 1]
```diff
@@ -30,18 +30,19 @@ var svg = d3.select("#timeline1").append("svg").attr("width", 500)
 
 Works with circles. In case the rectangular edges are too pointy.
 
-![Circular Timeline](https://raw.github.com/jiahuang/d3-timeline/master/examples/timeline2.png
-)
+![Circular Timeline](examples/timeline2.png)
+
+Combine rectangles and circles to your liking.
```
Comment (jiahuang) [inline README.md]: Can you add the example dataset in here? Or mention that you can add a "display" key to the data item?

Comment (inodb) [inline README.md]: Fixed in latest commit

**Comment by jiahuang** [comment]
:+1: Thanks!

**Pull request merged**